### PR TITLE
Add Host Check bypass to apache and nginx sample config

### PR DESCRIPTION
**Pull request by avrelaun**

### CODE REVIEW

File: users/reverseproxy.rst [hunk 1]
```diff
@@ -39,6 +40,7 @@ Nginx
 
     location /syncthing/ {
       proxy_set_header        Host $host;
+      #proxy_set_header        Host localhost; # Uncomment to bypass Host Check
```
Comment (canton7) [inline users/reverseproxy.rst]: Probably wants a comment saying to remove `proxy_set_header        Host $host;` if you do this?

**Comment by calmh** [comment]
This needs a note on the concerns associated with doing this, namely that you open yourself up for attacks that Synchting doesn't protect against when fooled, hence this is a bad idea unless the proxy is doing some sort of authentication of the user before passing the request onwards.

**Comment by avrelaun** [comment]
Is there a more secure way to access syncthing remotely ? I could add a link pointing to the security implications as detailed here: https://docs.syncthing.net/users/faq.html#why-do-i-get-host-check-error-in-the-gui-api

File: users/reverseproxy.rst (hunk 1 above)
Comment (avrelaun) [inline users/reverseproxy.rst]: see 2e83649

**Comment by calmh** [comment]
I think lying about the Host header is probably the wrong solution anyhow.

**Pull request closed without merging**